test/volume/org.xen.xapi.storage.dummy/sr.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+"""
+ Copyright (C) Citrix Systems, Inc.
+"""
+
+import json
+import os
+import os.path
```

Comment:
This is not needed. You are already importing os, and calling `os.path` explicitly below